src/main/java/org/opensearch/sdk/ExtensionsRunner.java

```diff
@@ -71,7 +71,8 @@ public class ExtensionsRunner {
     /**
      * This field is initialized by a call from {@link ExtensionsInitRequestHandler}.
      */
-    public TransportService extensionTransportService = null;
+    TransportService extensionTransportService = null;
```

Comment:
private here?

Reply:
I had left it package private thinking it was needed for tests, but looks like after the refactoring it no longer is, so I switched it to private.   Invalidated the two reviews, @owaiskazi19 and @joshpalis do you mind reviewing again? :)